pkg/scd/operational_intents_handler.go

```diff
@@ -384,6 +382,10 @@ func (a *Server) PutOperationalIntentReference(ctx context.Context, manager stri
 	if !state.IsValidInDSS() {
 		return nil, nil, stacktrace.NewErrorWithCode(dsserr.BadRequest, "Invalid OperationalIntent state: %s", params.State)
 	}
+	hasCMSARole := auth.HasScope(authorizedManager.Scopes, restapi.UtmConformanceMonitoringSaScope)
+	if state.IsOffNominal() && !hasCMSARole {
```

Comment:
nit: the thing we want to know here is probably `IsConformanceMonitoringState` or `IsCMSAState` or `RequiresCMSA` rather than `IsOffNominal` (though the logic needs no changes)